### ISSUE TITLE
Share options

### ISSUE THREAD

**Issue by severinmenard** [plepe/OpenStreetBrowser#66]
I did not find a share button (as eg in uMap or MapContrib) to get the possibility of getting:
- a link with both zoom level, coordinates of center of the map and displayed layers
- an HTML iframe to embed the map in a web page

**Comment by plepe** [comment]
1. Permalink: You can just copy the address from your addressbar, this contains all this information. Do you think an additional Permalink for copying is necessary?

2. No, something like this currently does not exist -> feature request.

**Comment by plepe** [comment]
Permalinks are now online, see https://blog.openstreetbrowser.org/node/60 . I hope it works as expected.

**Comment by IgorEliezer** [comment]
Out of curiosity, what does the permalink that you implemented do? I know what a permalink is for, but I click on it and nothing happens. Is it for mobile users?

**Comment by plepe** [comment]
Of course, it does nothing, because it represents the current state of the application. I could open OpenStreetBrowser in a new windows though, that might be an improvement.

I thought the Permalink is for copying (right-click) and sharing ...

The lang parameter is a good idea.
Any further ideas?

Ideas that I had: an interactive permalink-configurator where every parameter is explained and you can choose if you want to include it or not.

**Comment by IgorEliezer** [comment]
> Of course, it does nothing, because it represents the current state of the application. I could open OpenStreetBrowser in a new windows though, that might be an improvement.

I asked because it seemed it did nothing in comparison to, for instance, OpenRailwayMap: clicking on "permalink" will refresh the window and the address bar with the last state since the address bar doesn't update as I use/zoom/pan the map, a thing OpenStreetBrowser does seamlessly.

Alternatively to opening a new window, clicking on "Permalink" would copy the current state (lang, lat, lon, z, categories) to clipboard and a small pop-up or tool-tip "Copied to clipboard!" would appear just to let the user know what happened.
